tests/ClientTest.php

```diff
@@ -139,4 +140,80 @@ public function testGetTopLevelFolders() {
     $this->assertTrue(is_array($folder));
     $this->assertInstanceOf('cweagans\webdam\Entity\Folder', $folder[0]);
   }
+
+  /**
+   * Tests getPresignUrl.
+   */
+  public function testGetPresignUrl() {
+    $mock = new MockHandler([
+      new Response(200, [], '{"access_token":"ACCESS_TOKEN", "expires_in":3600, "token_type":"bearer", "refresh_token":"REFRESH_TOKEN"}'),
+      new Response(200, [], file_get_contents(__DIR__ . '/json/presign.json')),
+    ]);
+    $handler = HandlerStack::create($mock);
+    $guzzleClient = new GClient(['handler' => $handler]);
+    $client = new Client($guzzleClient, '', '', '', '');
+
+    $file_data = [
+      "contenttype" => "image/png",
+      "filename" => "not_for_real.png",
+      "filesize" => "664",
```

Comment:
Read the test image file into a variable in your test. Pass that var into the client and have the client figure out how big the file in that variable is. If it's easier to just pass in the filesize as an arg, that's okay too.